src/python_testing/TC_DA_1_9.py

```diff
@@ -0,0 +1,197 @@
+            TestStep(7, "Test commissioning with valid DAC and PAI",
+                     "Commissioning succeeds without any attestation errors"),
+        ]
+
+    @async_test_body
+    async def test_TC_DA_1_9(self):
+        test_vectors = [
+            {
+                'name': 'tc_dac_revoked',
+                'dac_provider': 'revoked-dac-01.json',
+                'revocation_set': 'revocation-set.json',
+                'expects_commissioning_success': False,
+                'expects_att_err': 302
+            },
+            {
+                'name': 'tc_pai_revoked',
+                'dac_provider': 'revoked-pai.json',
+                'revocation_set': 'revocation-set.json',
+                'expects_commissioning_success': False,
+                'expects_att_err': 202
+            },
+            {
+                'name': 'tc_dac_and_pai_revoked',
+                'dac_provider': 'revoked-dac-and-pai.json',
+                'revocation_set': 'revocation-set.json',
+                'expects_commissioning_success': False,
+                'expects_att_err': 208
+            },
+            {
+                'name': 'tc_dac_revoked_using_delegated_crl_signer',
+                'dac_provider': 'indirect-revoked-dac-01.json',
+                'revocation_set': 'revocation-set.json',
+                'expects_commissioning_success': False,
+                'expects_att_err': 302
+            },
+            {
+                'name': 'tc_pai_revoked_using_delegated_crl_signer',
+                'dac_provider': 'indirect-revoked-pai-03.json',
+                'revocation_set': 'revocation-set.json',
+                'expects_commissioning_success': False,
+                'expects_att_err': 202
+            },
+            {
+                'name': 'tc_dac_and_pai_revoked_using_delegated_crl_signer',
+                'dac_provider': 'indirect-revoked-dac-01-pai-03.json',
+                'revocation_set': 'revocation-set.json',
+                'expects_commissioning_success': False,
+                'expects_att_err': 208
+            },
+            {
+                'name': 'tc_dac_and_pai_valid',
+                'dac_provider': None,
+                'revocation_set': 'revocation-set.json',
+                'expects_commissioning_success': True,
+                'expects_att_err': None
+            }
+        ]
+
+        for idx, test_case in enumerate(test_vectors):
+            self.step(idx + 1)
+
+            # Clean up any existing KVS files
+            subprocess.call("rm -f /tmp/tmpkvs*", shell=True)
+
+            # Create log files for this test case
+            os.makedirs(self.app_log_path, exist_ok=True)
+            app_log_file_name = f"{self.app_log_path}/{test_case['name']}_app.log"
+
+            with open(app_log_file_name, 'w') as app_log_file:
+                # Start the all-clusters-app with appropriate DAC provider
+                app_args = '--trace_decode 1 --KVS /tmp/tmpkvs'
```

Comment:
Please do not directly refer to `/tmp` if possible. Use `all-clusters-kvs` in the current directory, for example.

Reply:
changed to `all-clusters-kvs`